anteform.p8

```diff
@@ -1234,38 +1187,40 @@ function animatespr(sprnum)
   fset(sprnum,2,true)
 end
 
+function print_r(str, x, ...)
+  print(str, x - 4 * #tostr(str), ...)
+end
+
 function draw_stats()
-  local linestart,midlinestart,longlinestart=106,110,119
+  local linestart,lineend=106,127
   print("cond",linestart,0,5)
-  print(band(hero.st,1)==1 and 'p' or 'g',125,0,6)
+  print_r(band(hero.st,1)==1 and 'p' or 'g',lineend,0,6)
   print("lvl",linestart,8,5)
-  print(hero.lvl,longlinestart,8,6)
+  print_r(hero. lvl,lineend,8,6)
   print("hp",linestart,16,5)
-  print(hero.hp,linestart+8,16,6)
+  print_r(hero.hp,lineend,16,6)
   print("ap",linestart,24,5)
-  print(hero.mp,linestart+8,24,6)
+  print_r(hero.mp,lineend,24,6)
   print("$",linestart,32,5)
-  print(hero.gp,midlinestart,32,6)
+  print_r(hero.gp,lineend,32,6)
   print("f",linestart,40,5)
-  print(hero.fd,midlinestart,40,6)
+  print_r(hero.fd,lineend,40,6)
   print("exp",linestart,48,5)
-  print(hero.exp,linestart,55,6)
+  print_r(hero.exp,lineend,55,6)
   print("dex",linestart,63,5)
-  print(hero.dex,longlinestart,63,6)
+  print_r(hero.dex,lineend,63,6)
   print("int",linestart,71,5)
-  print(hero.int,longlinestart,71,6)
+  print_r(hero.int,lineend,71,6)
   print("str",linestart,79,5)
-  print(hero.str,longlinestart,79,6)
+  print_r(hero.str,lineend,79,6)
```

Comment:
This is a functional change, both aesthetic and code-simplifying. I can easily undo the right aligning and approach the code reduction here another way.